public/styles/responsive.scss

```diff
@@ -18,79 +17,139 @@ $hover-black: rgba(0,0,0,0.2);
   .section {
      padding: 40px 0;
    } 
+
    .banner {
      position: relative;
      height: 45vh;
      background-size: cover;
      background-position: top;
    }
+
    .text-group--featured{
      margin-top: 120px;
      margin-left: 30px;
    }
+
    .navbar-burger {
      margin-right: 20px;
      padding: 15px;
    }
+
    .heading-primary-main {
       font-size: 20px !important;
   }
+
  .heading-primary-sub {
    font-size: 15px !important;
    }
+
    .btn-white {
      margin-top: 30px;
      margin-left: 10px;
      background-color: $orange-color;
      color: $white-color;
    }
+
+  .header {
+     position: relative;
+    height: 40vh;
+    background-size: cover;
+    background-position: top;
+  }
+
+  .featured-text{
+    margin-top: 30px;
+  }
+
+  .heading-primary-main {
+     font-size: 20px !important;
+  }
+
+  .heading-primary-sub {
+    font-size: 15px !important;
+    }
+
```

Comment:
These can be removed; `.header `has been changed to `.banner`

Reply:
Done @TheoOkafor .